src/solvers.jl

```diff
@@ -431,3 +431,16 @@ function merge_duplicates{CoefType,IntType<:Integer}(::Type{IntType},aff::Generi
     return indices, coeffs
 
 end
+
+# get the solve time
+function getSolveTime(m::Model)
+    if !m.internalModelLoaded
+        Base.warn("Model not solved")
+        return nothing
```

Comment:
I think it's more Julia-like (for type stability reasons) to throw an exception when the value isn't available instead of returning ``nothing``.